packages/transcript-view/src/transcript-view.ts

```diff
@@ -0,0 +1,383 @@
+              this.transcriptEntryTemplate(entry),
+            )}
+          </div>
+        </div>
+      </div>
+    `;
+  }
+
+  private get autoScrollButtonTemplate(): TemplateResult {
+    return html`
+      <button
+        @click=${this.enableAutoScroll}
+        class="auto-scroll-button"
+        style="display: ${this.autoScroll ? 'none' : 'inline-block'}"
+      >
+        Scroll text with audio
+      </button>
+    `;
+  }
+
+  private get timeDisplayTemplate(): TemplateResult {
+    return html`
+      <div class="time-display" style="top: ${this.timeScrollTop}px; display: ${this.timeDisplay}">
+        <duration-formatter .seconds=${this.currentEntryStartTime}> </duration-formatter>
+      </div>
+    `;
+  }
+
+  private transcriptEntryTemplate(entry: TranscriptEntryConfig): TemplateResult {
+    const currentEntryId = this.currentEntry ? this.currentEntry.id : -1;
+    const active = entry.id === currentEntryId;
+    const selected = entry.searchMatchIndex === this.selectedSearchResultIndex;
+    return html`
+      <transcript-entry
+        .entry=${entry}
+        ?isSelected=${selected}
+        ?isActive=${active}
+        data-search-result-index=${entry.searchMatchIndex}
+        @userSelected=${this.transcriptEntrySelected}
+      >
+      </transcript-entry>
+    `;
+  }
+
+  private get contextZoneDevTemplates(): TemplateResult {
+    return html`
+      ${this.topContextZoneDevTemplate} ${this.bottomContextZoneDevTemplate}
+    `;
+  }
+
+  private get topContextZoneDevTemplate(): TemplateResult {
+    return html`
+      <div class="top context-overlay" style="height: ${this.topContextHeight}px"></div>
+    `;
+  }
+
+  private get bottomContextZoneDevTemplate(): TemplateResult {
+    return html`
+      <div class="bottom context-overlay" style="height: ${this.bottomContextHeight}px"></div>
+    `;
+  }
+
+  get currentEntryStartTime(): number {
+    return this.currentEntry ? this.currentEntry.start : 0;
+  }
+
+  static get styles(): CSSResult {
+    const transcriptHeightCss = css`var(--transcriptHeight, 200px)`;
+
+    const timeColorCss = css`var(--timeColor, white)`;
+    const timeColumnWidthCss = css`var(--timeColumnWidth, 5rem)`;
+
+    const autoScrollButtonFontColorCss = css`var(--autoScrollButtonFontColor, black)`;
+    const autoScrollButtonBackgroundColorCss = css`var(--autoScrollButtonBackgroundColor, white)`;
+
+    return css`
+      .container {
+        position: relative;
+      }
+
+      .auto-scroll-button {
+        position: absolute;
+        left: 0;
+        right: 0;
+        margin: auto;
+        width: 8rem;
+        bottom: 1rem;
+        border-radius: 1rem;
+        border: 0;
+        color: ${autoScrollButtonFontColorCss};
+        background-color: ${autoScrollButtonBackgroundColorCss};
+      }
+
+      .context-overlay {
+        position: absolute;
+        left: 0;
+        width: 100%;
+        height: 0;
+        z-index: -1;
+        display: block;
```

Comment:
You can remove the redundant `display: block` property since setting `position` to `absolute` changes the element's display to `block` by default.